styles.css

```
@@ -1,31 +1,331 @@
 /* ------------------------------- 1. Global Styles -------------------------------- */
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+    text-decoration: none;
+}
+
+
+/* ------------------------------- 1.1 Colors ------------------------------------------ */
+
+:root {
+    --purple: #5722CD;
+    --blue:   #4845E4;
+    --pink:   #ED4385;
+    --text-color:   #0C0C0C;
+    --light-purple: #F2F0FF;
+    --light-gray:   #F3F6F9;
+    --gray:         #7A7A7A;
+    --gradient: radial-gradient(circle, #4845e4, #564de6, #6356e9, #6e5eeb, #7967ed);
+
+
+}
 
 /* ------------------------------- 2. Typography ----------------------------------- */
+h1,h2,h3,h4,h5,h6 {
+    font-family: "Merriweather", sans-serif;
+}
+body {
+    font-family: "Roboto", sans-serif;
+}
+
+a {
+    color: var(--purple);
+}
 
 /* ------------------------------- 3. Layout --------------------------------------- */
 
+
 /* --------------- 3.1 Reusables ----------------------- */
 
 /* --------------- 3.2 Other --------------------------- */
 
 /* ------------------------------- 4. Areas ---------------------------------------- */
 
 /* --------------- 4.1 Header ------------------------- */
+header {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    color: white;
+    background-image: var(--gradient);
+}
+
+header a {
+    color: inherit;
+}
+
+header h1 {
+    font-size: 36px;
+}
+
+.header-nav {
+    align-self: flex-end;
+}
+
+.header-nav ul {
+    display: flex;
+    gap: 30px;
+    font-size: 16px;
+    font-weight: 300;
+    padding: 25px;
+    margin-right: 100px;
+}
+
+.header-cta {
+    display: flex;
+    flex-direction: column;
+    padding: 30px;
+    justify-content: center;
+    text-align: center;
+    align-items: center;
+    gap: 30px;
+    width: 75%;
+}
+
+.header-cta p {
+    font-size: 22px;
+    font-weight: lighter;
+}
+
+.header-button-cta {
+    color: white;
+    padding: 20px;
+    width: 224px;
+    border-radius: 26px;
+    border: none;
+    background-color: var(--pink);
+    box-shadow: rgba(0, 0, 0, 0.35) 0 5px 15px;
+    text-transform: uppercase;
+
+}
+
+.header-icon {
+    margin-top: 75px;
+    width: 75px;
+    height: 100px;
+    background: url("./assets/squiggle_line_stripe.svg") no-repeat ;
+}
 
 /* --------------- 4.2 Introduction ------------------- */
 
+.business-flex-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 30px;
+    text-align: center;
+    margin-top: 80px;
+}
+
+.business-flex-container h2 {
+    color: var(--purple);
+}
+
+.business-flex-container p {
+    margin-top: 45px;
+}
+
+/* --------------- 4.3 Work Section ------------------- */
+.work-section {
+    margin-top: 65px;
+    background-color: var(--light-gray);
+}
+
+.work-collection-container {
+    display: flex;
+    flex-direction: column;
+    margin: 0 100px 0 100px;
```

Dit kun je korter schrijven
```suggestion
    margin: 0 100px;
```